src/utils/popup-keyboard-accessibility.tsx

```diff
@@ -25,6 +27,15 @@ export const withKeyboardA11y = (WrappedComponent): any =>
       this.focusOnDefault();
     }
 
+    /**
+     * checks if the overlay is open
+     * @returns {boolean} - true if the overlay is open
+     * @memberof HOC
+     */
+    get isOverlayOpen() {
+      return this.props.player.ui.store.getState().overlay.isOpen;
```

Comment:
why not just mapStateToProps ?

Reply:
I don't want all components wrapped in HOC be re-rendered every time any of plugin open/close Overlay